src/renderer/components/ft-list-dropdown/ft-list-dropdown.js

```diff
@@ -31,5 +31,36 @@ export default Vue.extend({
     listType: function () {
       return this.$store.getters.getListType
     }
+  },
+  methods: {
+    removeWhitespace,
+    handleIconKeyPress(event) {
+      if (event instanceof KeyboardEvent) {
+        if (event.key === 'Tab') {
+          return
+        }
+
+        event.preventDefault()
+
+        if (event.key !== 'Enter' && event.key !== ' ' && event.key !== 'ArrowDown') {
+          return
+        }
+      }
+
+      const firstOption = document.getElementById('buttonOption0')
+      firstOption.setAttribute('tabIndex', 1)
+      firstOption.focus()
+    },
```

Comment:
This can be handled a lot cleaner and with less code by filtering the keys on the vue side.